collections/_posts/2023-07-06-Mean_teachers_are_better_role_models.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: review
+title: Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results
+tags: machine-learning, neural and evolutionary computing
+author: Tarvainen Antti, Valpola Harri
```

Comment:
The `author` field here is to indicate the author of the review (i.e. you), not the authors of the paper :wink: 
```suggestion
author: "Morgane des Ligneris"
```